pkgs/development/compilers/rust/rustc.nix

```diff
@@ -139,6 +139,8 @@ in stdenv.mkDerivation (finalAttrs: {
     "${setBuild}.llvm-config=${llvmSharedForBuild.dev}/bin/llvm-config"
     "${setHost}.llvm-config=${llvmSharedForHost.dev}/bin/llvm-config"
     "${setTarget}.llvm-config=${llvmSharedForTarget.dev}/bin/llvm-config"
+  ] ++ optionals fastCross [
+    "--disable-llvm-bitcode-linker"
```

Comment:
The impact of this is that no whole program optimization can be done?

Reply:
Some comment in the code would be nice to have because it's not obvious why we are doing this.

Reply:
The purpose of fastCross is to only build the standard library, not a whole compiler, for the cross target.  There's therefore no point building a linker (which is only used in some obscure nvidia target anyway as far as I can tell.)

Reply:
I feel like it's a bit redundant with the comment explaining fastCross above, but I've added another comment.

Reply:
Thanks!